templates/candidates-single.html

```diff
@@ -64,9 +64,9 @@ <h5 class="entity__term__label">Elections</h5>
     <div class="page-controls__bottom">
       <div class="container">
          <nav class="page-tabs">
-            <ul role="tablist">
-              <li role="presentation" class="page-tabs__item"><a role="tab" tabindex="0" aria-controls="panel1" aria-selected="true" href="#section-1">Financial Summary</a></li>
-              <li role="presentation" class="page-tabs__item"><a role="tab" tabindex="-1" aria-controls="panel2" href="#section-2">Filings</a></li>
+            <ul role="tablist" name="tabs">
```

Comment:
`name` shouldn't be used on `<ul>` and `<li>`s . Probably should be `data-name=` ?